src/videojs-hls.js

```diff
@@ -450,6 +446,8 @@ videojs.Hls.prototype.setCurrentTime = function(currentTime) {
     return currentTime;
   }
 
+  this.lastSegmentLoaded_ = null;
```

Comment:
I still think we can get rid of this additional instance variable but that explanation makes sense if you allow that it should exist in the first place :)